### PR TITLE
  Create plug that loads the current tenant assign from a given header…

### PR DESCRIPTION
**Pull request by t00lmaker**
Create plug hat loads the current tenant assign from a given header…

### CODE REVIEW

File: package-test/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (kelvinst) [inline package-test/LICENSE]: Can you remove this whole folder and amend it to the previous commit? We don't need this. :)

File: lib/mix/triplex.ex [hunk 2]
```diff
@@ -22,7 +22,7 @@ defmodule Mix.Triplex do
   @doc """
   Returns the path for your tenant migrations.
   """
-  def migrations_path(repo \\ config().repo)
+  #def migrations_path(repo \\ config().repo)
```
Comment (kelvinst) [inline lib/mix/triplex.ex]: Rollback this comment.

```suggestion
  def migrations_path(repo \\ config().repo)
```

**Comment by mgiacomini** [comment]
@t00lmaker could you resolve the mix.exs conflicts before we merge this pr?

**Comment by kelvinst** [comment]
Thanks for the PR! We will review and merge it soon!

Also, this will be released on version 1.7, which we will also extract a `triplex_plugs` lib with the plugs there, so triplex will no longer have these here.

**Comment by GusGA** [comment]
This PR could be helpful to others and looks abandoned, Can I fork it and fix the issues so could be merged?

**Comment by kelvinst** [comment]
@GusGA we are moving all the plugs from this repo to another one, and we will probably move this one altogether. @mgiacomini please do not forget to copy the plug of this PR to that new `triplex_plugs` project.

**Comment by gusgamdyo** [comment]
Excellent news @kelvinst, thanks for your effort.

**Comment by kelvinst** [comment]
Hey @mgiacomini, let's merge this to master before separating the plugs on another repository. SO that way @t00lmaker takes the credit for it at least. 😄 Thanks BTW.

**Comment by mgiacomini** [comment]
> Hey @mgiacomini, let's merge this to master before separating the plugs on another repository. SO that way @t00lmaker takes the credit for it at least. 😄 Thanks BTW.

Sure!

**Pull request closed without merging**